addons/frag/functions/fnc_dev_addRound.sqf

```diff
@@ -0,0 +1,98 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * This function adds a round to be traced
+ *
+ * Arguments:
+ * 0: Projectile <OBJECT>
+ * 1: Add projectile event handlers <BOOL>
+ * 2: Is the round blue <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_proj, false, false] call ace_frag_dev_addRound;
+ *
+ * Public: No
+ */
+params [
+    "_proj",
+    ["_addEHs", true, [true]],
+    ["_sidePlayer", true, [true]]
+];
+
+// track round on each frame
+// Create entry in position array from hashmap
+if (_sidePlayer) then
+{
```

Comment:
```suggestion
if (_sidePlayer) then {
```

Follow the [coding guidelines](https://ace3.acemod.org/wiki/development/coding-guidelines#51-braces-placement) in all files.